tests/search/attributesearch/cased/cased.sd

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+search cased {
```

Comment:
Consider renaming to _test.sd_. See comment above.

Reply:
Fixed